src/main/java/org/mule/extension/ftp/api/FtpFileMatcher.java

```diff
@@ -186,23 +190,62 @@ public LocalDateTime getTimestampSince() {
     return timestampSince;
   }
 
+  public void setTimestampSince(LocalDateTime timestampSince) {
+    this.timestampSince = timestampSince;
+  }
+
+
   public LocalDateTime getTimestampUntil() {
     return timestampUntil;
   }
 
+  public void setTimestampUntil(LocalDateTime timestampUntil) {
+    this.timestampUntil = timestampUntil;
+  }
+
   public TimeUnit getTimeUnit() {
     return timeUnit;
   }
 
+  public void setTimeUnit(TimeUnit timeUnit) {
+    this.timeUnit = timeUnit;
+  }
+
   public Long getUpdatedInTheLast() {
     return updatedInTheLast;
   }
 
+  public void setUpdatedInTheLast(Long updatedInTheLast) {
+    this.updatedInTheLast = updatedInTheLast;
+  }
+
   public Long getNotUpdatedInTheLast() {
     return notUpdatedInTheLast;
   }
 
+  public void setNotUpdatedInTheLast(Long notUpdatedInTheLast) {
+    this.notUpdatedInTheLast = notUpdatedInTheLast;
+  }
+
   public boolean isCaseSensitive() {
```

Comment:
The getters should be named all get<attribute>. PLease analyze the current getters to match this format

Reply:
Yes, but those getters names were already there and I noticed that are related to the FileMatcher documentation. Also that happened with other boolean values. I only added setters for those